sdk/playwrighttesting/arm-playwrighttesting/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+# Release History
+    
+## 1.0.0-beta.1 (2023-09-22)
+
+The package of @azure/arm-playwrighttesting is using our next generation design principles. To learn more, please refer to our documentation [Quick Start](https://aka.ms/js-track2-quickstart).
```

Comment:
Another minor issue to consider,  we are suggested not to use track2 for public documents, so you may consider update the name of aka link here:

[Quick Start](https://aka.ms/js-track2-quickstart).

Reply:
We will need to update the change log tool as well after @lirenhe  provide the right link to us. so I think it's okay to skip it here, we can update all the link later. how do you think ?